python/src/LevelSet_doc.i.in

```diff
@@ -41,14 +41,14 @@ Examples
 
```

Comment:
Can't suggest it using the GitHub interface, but line 40 should be changed to :

"Return the intersection between this LevelSet and another one as a LevelSet.

Reply:
Or something else, but the current sentence has incorrect grammar.